### ISSUE TITLE
how to use ORD datasets into the model?

### ISSUE THREAD

**Issue by ALR-alr** [shizhediao/DaVinci#7]
ORD datasets have two parts: scene-graph and region-description,. How can you use it to complete a I2T or T2I task? Do you rewrite the region descriptions to a whole sentence as a text input?

**Comment by shizhediao** [comment]
Yes, we did some data augmentation based on the original data. You can find the details in Appendix A.3 https://arxiv.org/pdf/2206.07699

**Comment by ALR-alr** [comment]
Thanks for your rapid reply! Whether the prompt template only consider the objects, but ignore the attributions and relations?

**Comment by shizhediao** [comment]
Yes, but you could explore it if you think it is beneficial.
Thanks!

**Comment by ALR-alr** [comment]
Thank you very much,  your reply is crucial to me!
